lib/fluent/plugin/in_tail.rb

```diff
@@ -1220,6 +1224,7 @@ def handle_notify
 
                     if group_watcher_limit || limit_bytes_per_second_reached? || should_shutdown_now?
                       # Just get out from tailing loop.
+                      @metrics.throttled.inc
```

Comment:
`should_shutdown_now?` isn't related with throttling so it should be treated separately.